<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>915</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>13</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Credit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 139, 87);</string>
  </property>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(135, 206, 235);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_mounhly_payment_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_overpayment_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_total_payment_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>419</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_mounhly_payment">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(143, 188, 143);</string>
      </property>
      <property name="text">
       <string>Ежемесячный 
платёж</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_overpayment">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(143, 188, 143);</string>
      </property>
      <property name="text">
       <string>Переплата по 
кредиту</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_total_payment">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(143, 188, 143);</string>
      </property>
      <property name="text">
       <string>Общая
выплата</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_diff">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(143, 188, 143);</string>
        </property>
        <property name="text">
         <string>дифференцированный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_annuity">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(143, 188, 143);</string>
        </property>
        <property name="text">
         <string>аннуитетный</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_amount_credit">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(143, 188, 143);</string>
        </property>
        <property name="text">
         <string>Сумма 
кредита</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_term">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(143, 188, 143);</string>
        </property>
        <property name="text">
         <string>Срок</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_interest_rate">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(143, 188, 143);</string>
        </property>
        <property name="text">
         <string>Процентная 
ставка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_amount_credit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>1000000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_term">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>12</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_interest_rate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_diff_m_payment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>9</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>80</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_mounhly_payment_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 188, 143);</string>
    </property>
    <property name="text">
     <string>Платёж за 
первый месяц</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_first_month">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>220</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_mounhly_payment_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>220</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 188, 143);</string>
    </property>
    <property name="text">
     <string>Платёж за 
последний месяц</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_last_month">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>220</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="midLineWidth">
     <number>-3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_diff">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
